postgres/toddl.go

```diff
@@ -28,7 +28,7 @@ import (
 // Spanner. It uses the source schema in conv.SrcSchema, and writes
 // the Spanner schema to conv.SpSchema.
 func schemaToDDL(conv *internal.Conv) error {
-	schemaForeignKeys := make(map[string]bool)
+	spKeyNames := make(map[string]bool)
```

Comment:
Suggest same name change (usedNames) and added comment as in mysql/toddl.go.

Reply:
updated as mentioned